<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="lcdNumAcc">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumCmdReg">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>50</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="accLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>111</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Akkumulator</string>
    </property>
   </widget>
   <widget class="QLabel" name="crLabel">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>121</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Befehlsregister</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="verticalScrollBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>16</width>
      <height>160</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="runButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>100</y>
      <width>109</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string>Durchlauf</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stepButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>150</y>
      <width>109</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string>Schritt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>200</y>
      <width>109</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>300</y>
      <width>109</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string>Speichern</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>250</y>
      <width>109</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string>Laden</string>
    </property>
   </widget>
   <widget class="QListView" name="reglistView">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>291</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="helpButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>350</y>
      <width>109</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string>Hilfe</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>715</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
